src/handlers.ts

```diff
@@ -121,6 +131,39 @@ export async function postPublishedWebhook(
     });
 }
 
+export async function siteChangedWebhook(
+    ctx: Context<{ Variables: HonoContextVariables }>,
+    next: Next,
+) {
+    // Retrieve site settings from Ghost
+    const host = ctx.req.header('host')
+
+    const settings = await ky
+        .get(`https://${host}/ghost/api/admin/site/`)
+        .json<GhostSiteSettings>();
```

Comment:
This is sus, but not sure what the best thing to do here is as we don't know anything about the requesting client (so am just assuming that the request came from the Ghost instance on that is accessible on the same host 🙃)